RELEASE/scripts/sl_ascend.ash

```diff
@@ -13909,9 +13958,28 @@ boolean sl_tavern()
 			}
 		}
 
+		if((my_path() != "Actually Ed the Undying") && (monster_level_adjustment() <= 299))
+		{
+			// Maximize ML First by using equipment
+			if(useMaximizeToEquip())
+			{
+				addToMaximize("1000ml 300max");
+			}
+
+			// Asdon usage increases Rat King chance by 8.3%
+			if(monster_level_adjustment() <= 299)
+			{
+				asdonBuff($effect[Driving Recklessly]);
+			}
+		}
+
+		//Turn up the MCD
+		sl_change_mcd(11);
+
 		if(!maximized)
 		{
-			addToMaximize("200cold damage 20max,200hot damage 20max,200spooky damage 20max,200stench damage 20max,100ml 149max");
+			// Tails are a better time saving investment
+			addToMaximize("200cold damage 20max,200hot damage 20max,200spooky damage 20max,200stench damage 20max,1000ml 300max");
```

Comment:
I still don't think it's worth going over 150 ML for tails.

Reply:
Fair enough. I did notice that the more I toggle this number the more erratic it was, sometimes not even getting 1, granted if I maxxed it I'd get 4-6....which is overkill. Due to the erraticness of it I didn't settle on the perfect number. At ML 150 it is pretty much "guaranteed 1". If the player has olfact or a pseudo-olfact, this should be sufficient to gather enough ratchets before the control room unlocks.